api/src/pcapi/core/fraud/api/ubble.py

```diff
@@ -68,3 +73,11 @@ def get_ubble_fraud_check(identification_id: str) -> typing.Optional[fraud_model
         .one_or_none()
     )
     return fraud_check
+
+
+def does_match_ubble_test_email(email: str) -> typing.Optional[re.Match]:
+    # This function MUST NEVER return an other value than None in production environment
```

Comment:
(détail) le commentaire me semblerait plus facile à lire sans la double négation

```suggestion
    # This function MUST ALWAYS return None in production environment
```

Reply:
très vrai, je corrige ça avant de merger